src/cleaner/__init__.py

```diff
@@ -174,18 +188,26 @@ def process_single_case(
 
         # Initialize cleaned output data
         output_json_data = {
-            "case_number": input_dict["code"],
-            "attorney_type": input_dict["party information"]["appointed or retained"],
-            "county": input_dict["county"],
-            "html_hash": input_dict["html_hash"],
-            "charges": [],
-            "earliest_charge_date": "",
-            "motions": [],
-            "has_evidence_of_representation": False,
-            "defense_attorney": self.hash_defense_attorney(input_dict),
             "parsing_date": dt.datetime.today().strftime("%Y-%m-%d"),
+            "html_hash": input_dict["html_hash"],
+            "Case Metadata": {
+                "county": input_dict["Case Metadata"]["county"]
+            },            
+            "Defendant Information": {
+                "appointed_or_retained": input_dict["Defendent Information"]["appointed or retained"],
+                "defense_attorney": self.hash_defense_attorney(input_dict),
+            },
+            "Charge Information": [],
+            "Case Details": {
+                "earliest_charge_date": "",
+                "has_evidence_of_representation": False,
```

Comment:
I wonder if we should just omit case details and charge information from this initializing record, then append the values later.  I wonder if there's a chance to get accidental false negatives here.